tests/unit/legal-api-mixin.spec.ts

```diff
@@ -0,0 +1,281 @@
+import Vue from 'vue'
```

Comment:
The main motivator here was to test all code paths and preempt low codecov numbers.

Reply:
nice.  very clean tests.  Haven't used sinon much so good bit of learning for me reading these tests!